### ISSUE TITLE
Failed to load the `console-app` service page for sample `StandaloneDashboard`

### ISSUE THREAD

**Issue by Menghua1** [dotnet/aspire-samples#597]
**Describe the issue:**
After running the `azd up` command, click the `console-app` service endpoint in the console output log, and the page fails to load, as shown below:
![Image](https://github.com/user-attachments/assets/423de5f4-f8da-4aae-bfe1-dfb840d5bd77)
More error information about the container on the azure portal is as follows:
![Image](https://github.com/user-attachments/assets/f1a3d73e-32c3-4726-a8a9-a29234d8d169)
![Image](https://github.com/user-attachments/assets/76c1aef3-a0b8-4d4c-a846-d7c4d66dbec3)
![Image](https://github.com/user-attachments/assets/7d6fcc83-e8cb-46ce-b1d2-eb63e5705d47)

**Repro Steps:**

1. Clone code.
2. Run command `cd samples/StandaloneDashboard`.
3. Run command `azd init`.
4. Run command `azd up`.
5. Click the `console-app` service endpoint.

**Environment:**

- Azd version: azd version 1.12.0-beta.1-daily.4359062 (commit https://github.com/Azure/azure-dev/commit/6cd82ae9ff02e156a0056e994215191469442ab3).
- Aspire version: `9.0.0`.
- Sample: [StandaloneDashboard](https://github.com/dotnet/aspire-samples/tree/main/samples/StandaloneDashboard).
- Dotnet SDK version: `9.0.100`.
- Branch: main.

**Expected behavior:**
The `console-app` service page can be loaded successfully.

@rajeshkamal5050, @vhvb1989 for notification.

**Comment by Menghua1** [comment]
@rajeshkamal5050 In the latest round of azd manual testing, this issue still exists.

- Sample: [StandaloneDashboard](https://github.com/dotnet/aspire-samples/tree/main/samples/StandaloneDashboard).
- Branch: main.
- Dotnet SDK version: 9.0.101
- OS: Windows.
- Azd version: azd version 1.12.0-beta.1-daily.4442669 (commit 9b60cbc2a703200eb04d325988dc359886b2dfcb).

**Comment by davidfowl** [comment]
This sample isn’t deployable

**Comment by Menghua1** [comment]
@rajeshkamal5050 In the latest round of azd manual testing, this issue still exists.

- Sample: [StandaloneDashboard](https://github.com/dotnet/aspire-samples/tree/main/samples/StandaloneDashboard).
- Branch: main.
- Dotnet SDK version: 9.0.102
- OS: Windows.
- Azd version: azd version 1.12.0-beta.1-daily.4489231 (commit a6eb17988fc047579a8edaaa86f302f5f0ab4bb8).

**Comment by Menghua1** [comment]
@rajeshkamal5050 In the latest round of azd manual testing, this issue still exists.

- Sample: [StandaloneDashboard](https://github.com/dotnet/aspire-samples/tree/main/samples/StandaloneDashboard).
- Branch: main.
- Dotnet SDK version: 9.0.102
- OS: Windows.
- Azd version: azd version 1.13.0-beta.1-daily.4578104 (commit a1d3f562baf3c4035accb4a64690542db5006126).

**Comment by davidfowl** [comment]
@Menghua1 this sample will never be deployable with azd. It's not an aspire sample

**Comment by davidfowl** [comment]
It is.

**Comment by vhvb1989** [comment]
Yeah, no need to validate this specific sample with azd. I wonder if it should even be removed from the samples repo @davidfowl .  Is it just for local experience? don't we get that already from `dotnet new aspire-starter` ?

**Comment by davidfowl** [comment]
This sample shows how to use the standalone dashboard with a C# console application
